VRCFaceTracking/Params/Expressions/Legacy/Eye/EyeTrackingParams.cs

```diff
@@ -0,0 +1,148 @@
+                    return exp.Shapes[(int)UnifiedExpressions.EyeSquintLeft].Weight;
+                return exp.Eye.Left.Openness;
+            }, "LeftEyeLidExpandedSqueeze"),
+            
+            new BinaryParameter(exp =>
+            {
+                if (exp.Shapes[(int)UnifiedExpressions.EyeWideRight].Weight > 0)
+                    return exp.Shapes[(int)UnifiedExpressions.EyeWideRight].Weight; 
+                if (exp.Shapes[(int)UnifiedExpressions.EyeSquintRight].Weight > 0)
+                    return exp.Shapes[(int)UnifiedExpressions.EyeSquintRight].Weight;
+                return exp.Eye.Right.Openness;
+            }, "RightEyeLidExpandedSqueeze"),
+            
+            new BinaryParameter(exp =>
+            {
+                if (exp.Shapes[(int)UnifiedExpressions.EyeWideLeft].Weight + exp.Shapes[(int)UnifiedExpressions.EyeWideRight].Weight / 2.0f > 0)
+                    return exp.Shapes[(int)UnifiedExpressions.EyeWideLeft].Weight + exp.Shapes[(int)UnifiedExpressions.EyeWideRight].Weight / 2.0f; 
+                if (exp.Shapes[(int)UnifiedExpressions.EyeSquintRight].Weight + exp.Shapes[(int)UnifiedExpressions.EyeSquintLeft].Weight / 2.0f > 0)
+                    return exp.Shapes[(int)UnifiedExpressions.EyeSquintRight].Weight + exp.Shapes[(int)UnifiedExpressions.EyeSquintLeft].Weight / 2.0f;
+                return exp.Eye.Combined().Openness;
+            }, "CombinedEyeLidExpandedSqueeze"),
+            
+            #endregion
+
+            #region EyeLidExpandedSupplemental
+
+            // These parameters are used to distinguish when EyeLidExpanded / EyeLidExpandedSqueeze
+            // is returning a value as a Widen or Squeeze. Intended for the Bool or Binary param variant.
+            new BoolParameter(exp => exp.Shapes[(int)UnifiedExpressions.EyeWideLeft].Weight > 0, "LeftEyeWidenToggle"),
+            new BoolParameter(exp => exp.Shapes[(int)UnifiedExpressions.EyeWideRight].Weight > 0, "RightEyeWidenToggle"),
+            new BoolParameter(exp => exp.Shapes[(int)UnifiedExpressions.EyeWideLeft].Weight > exp.Shapes[(int)UnifiedExpressions.EyeWideRight].Weight
+                ? exp.Shapes[(int)UnifiedExpressions.EyeWideLeft].Weight > 0
+                : exp.Shapes[(int)UnifiedExpressions.EyeWideRight].Weight > 0
+                , "EyesWidenToggle"),
+
+            new BoolParameter(exp => exp.Shapes[(int)UnifiedExpressions.EyeSquintLeft].Weight > 0, "RightEyeSqueezeToggle"),
+            new BoolParameter(exp => exp.Shapes[(int)UnifiedExpressions.EyeSquintRight].Weight > 0, "RightEyeSqueezeToggle"),
+            new BoolParameter(exp => exp.Shapes[(int)UnifiedExpressions.EyeSquintLeft].Weight > exp.Shapes[(int)UnifiedExpressions.EyeSquintRight].Weight
+                ? exp.Shapes[(int)UnifiedExpressions.EyeSquintLeft].Weight > 0
+                : exp.Shapes[(int)UnifiedExpressions.EyeSquintRight].Weight > 0
+                , "EyesSquintToggle"),
```

Comment:
Squint and squeeze are not the same thing when tracked. This should not be included with the old eye tracking bool switch

Reply:
>Squint should not be included with the old eye tracking bool switch

Oops looks like I accidentally renamed the output of this param. 

> Squint and squeeze are not the same thing when tracked.

As for Squint and squeeze not being the same I have elaborated on that in another comment thread.